Library/Homebrew/requirements/cctools_requirement.rb

```diff
@@ -0,0 +1,8 @@
+class CctoolsRequirement < Requirement
+  fatal true
+  default_formula 'cctools'
+
+  satisfy do
+    MacOS::Xcode.installed? || MacOS::CLT.installed? || Dir.exists?('/usr/local/opt/cctools')
```

Comment:
You can leave the last part off - should be fine letting it proceed to the normal mechanism for determining the formula is installed.